petric.py

```diff
@@ -141,7 +141,7 @@ def keys(self):
 
 class MetricsWithTimeout(cil_callbacks.Callback):
     """Stops the algorithm after `seconds`"""
-    def __init__(self, seconds=600, outdir=OUTDIR, transverse_slice=None, coronal_slice=None, **kwargs):
+    def __init__(self, seconds=600, outdir=OUTDIR, transverse_slice=None, coronal_slice=None, sagittal_slice=None, **kwargs):
```

Comment:
but this is an unused argument... do you want to plot sagittal images in TensorBoard too?

Reply:
moved to #104